modules_basic/timestamp.mcss

```diff
@@ -1,5 +1,6 @@
 Timestamp {
   a {
-
+    font-size: .8rem
+    color: gray
```

Comment:
hmm can we start with a simple colorscheme already. I think mcss allows mixins which would be ideal for this sort of thing right ? 
Is there a good file we can put all the colors in? that would be a themers dream!